salt/_states/metalk8s_cordon.py

```diff
@@ -70,4 +69,26 @@ def node_uncordoned(name, **kwargs):
     name
         the name of the node
     '''
+    # k-c-m 1.12 adds taint `node.kubernetes.io/unschedulable` when cordonning
```

Comment:
Oh wait you should do it in the module not in the state
`salt/_modules/metalk8s_cordon.py`